sonar-batch/src/main/java/org/sonar/batch/qualitygate/QualityGateVerifier.java

```diff
@@ -160,10 +160,11 @@ private String getAlertLabel(Resource project, ResolvedCondition condition, Metr
       .append(" ").append(operatorLabel(condition.operator())).append(" ")
       .append(alertValue(condition, level));
 
-    if (alertPeriod != null) {
-      Snapshot snapshot = resourceCache.get(project).snapshot();
-      stringBuilder.append(" ").append(periods.label(snapshot, alertPeriod));
-    }
+    // TODO Disabled because snapshot is no more created by the batch, but should be reactivated when the decorator will be moved to the batch
+    // if (alertPeriod != null) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3ACommentedOutCodeLine)